lib/internal/util/comparisons.js

```diff
@@ -5,30 +5,53 @@ const { isArrayBufferView } = require('internal/util/types');
 const { internalBinding } = require('internal/bootstrap/loaders');
 const { isDate, isMap, isRegExp, isSet } = internalBinding('types');
 
-function objectToString(o) {
-  return Object.prototype.toString.call(o);
+const ReflectApply = Reflect.apply;
```

Comment:
FWIW as a reviewer I'd still expect the "tamper safe"ing to be in a different PR. That said, changing it now here would be very frustrating to do likely so I'm fine with it here - just future personal preference.

Reply:
I'll see if I move those out due to this becoming semver-major.